src/ow1/heroes/mccree.opy

```diff
@@ -42,3 +42,21 @@ rule "[mccree.opy]: Stun flashed enemy":
     @Condition attacker.getCurrentHero() == Hero.MCCREE or attacker.getCurrentHero() == Hero.GENJI
 
     victim.setStatusEffect(eventPlayer, Status.STUNNED, OW1_MCCREE_FLASHBANG_STUN_DURATION)
+
+rule "[mccree.opy]: Remove damage reduction during roll":
+    @Event eachPlayer
+    @Hero mccree
+    @Condition eventPlayer.isUsingAbility1() == true
+
+    eventPlayer.setDamageReceived(100/(100-OW2_CASSIDY_ROLL_DAMAGE_REDUCTION)*100)
```

Comment:
```suggestion
    eventPlayer.setDamageReceived(percent(100/(100-OW2_CASSIDY_ROLL_DAMAGE_REDUCTION)))
```

the percent() function is defined in utilities.opy so we don't have to write *100 in the source code.